### PR TITLE
Add a new app forms tutorial

### PR DESCRIPTION
**Pull request by smbuthia**
A new tutorial that shows how to develop CHT application forms

Issue #276

### CODE REVIEW

File: content/en/apps/tutorials/app-forms.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+project-name
+  forms
+    app
+      assessment.xlsx
+```
+
+To convert and upload the form to your local instance, run the following command from the root folder:
+
+```zsh
+medic-conf --url=https://<username>:<password>@localhost --accept-self-signed-certs convert-app-forms upload-app-forms -- assessment.xlsx
```
Comment (kitsao) [inline content/en/apps/tutorials/app-forms.md:118]: Should be `http`?

File: content/en/apps/tutorials/app-forms.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+Add the following column names and rows to the settings sheet:
+
+| form_title     | form_id    | version | style | path | instance_name  | default_language  |
+| -------------- | ---------- | ------- | ----- | ---- | -------------- | ----------------- |
+| Assess patient | assessment | 1       | pages | data |                | en                |
+
+### 4. Convert the XLSForm and Upload the XForm
+
+Add the file to the `forms\app` subfolder in your project.
```
Comment (kitsao) [inline content/en/apps/tutorials/app-forms.md:106]: `forms/app`

File: content/en/apps/tutorials/app-forms.md (hunk 1 above)
Comment (kitsao) [inline content/en/apps/tutorials/app-forms.md:118]: `http`?
`-- assessment`, no extension.

File: content/en/apps/tutorials/app-forms.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+| begin group | parent          | Parent                      |          |                   |            |            |                     |                        |                |                               |         |
+| hidden      | _id             | Family UUID                 |          |                   |            |            |                     |                        |                |                               |         |
+| begin group | parent          | Grandparent                 |          |                   |            |            |                     |                        |                |                               |         |
+| hidden      | _id             | CHW Area UUID               |          |                   |            |            |                     |                        |                |                               |         |
+| hidden      | name            | CHW Name                    |          |                   |            |            |                     |                        |                |                               |         |
```
Comment (kitsao) [inline content/en/apps/tutorials/app-forms.md:63]: Missed `| begin group | contact          | CHW  |` that has CHW name and phone.

File: content/en/apps/tutorials/app-forms.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+```zsh
+medic-conf --url=https://<username>:<password>@localhost --accept-self-signed-certs convert-app-forms upload-app-forms -- assessment.xlsx
+```
+
+{{% alert title="Note" %}} Be sure to replace the values `<username>` and `<password>` with the actual username and password of your test instance. {{% /alert %}}
+
+## Next steps
+
+In the next tutorial, you will define the form `.properties.json` which will allow you to define the form’s title and icon, as well as when and where the form should be available.
```
Comment (kitsao) [inline content/en/apps/tutorials/app-forms.md:125]: `<form_id>.properties.json` is better.

File: content/en/apps/tutorials/app-forms.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+---
+
+App forms allow users to submit reports from Android devices
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to build App forms for CHT applications, including:
+
+- Authoring forms in Excel or Google sheets
```
Comment (kitsao) [inline content/en/apps/tutorials/app-forms.md:22]: ...or other spreadsheet applications.

**Review by kitsao — CHANGES_REQUESTED** [review]
Comments inline.

File: content/en/apps/tutorials/app-forms.md (hunk 3 above)
Comment (smbuthia) [reply content/en/apps/tutorials/app-forms.md:63]: Intentionally left them out to try and reduce the number of rows and simplify the table.

File: content/en/apps/tutorials/app-forms.md (hunk 5 above)
Comment (smbuthia) [reply content/en/apps/tutorials/app-forms.md]: Updated.

File: content/en/apps/tutorials/app-forms.md (hunk 4 above)
Comment (smbuthia) [reply content/en/apps/tutorials/app-forms.md]: Updated.

File: content/en/apps/tutorials/app-forms.md (hunk 1 above)
Comment (smbuthia) [reply content/en/apps/tutorials/app-forms.md]: Corrected.

File: content/en/apps/tutorials/app-forms.md (hunk 2 above)
Comment (smbuthia) [reply content/en/apps/tutorials/app-forms.md]: Corrected.

**Review by kitsao — APPROVED** [review]
Approved.

**Pull request merged**